### ISSUE TITLE
apt upgrade breaks IRkernel install with rocker_scripts/install_jupyter.sh

### ISSUE THREAD

**Issue by breiter** [rocker-org/rocker-versioned2#681]
### Container image name

rocker/rstudio:4

### What operating system are you seeing the problem on?

Linux, macOS

### System information

- Docker Desktop on macOS x64, Docker version 24.0.2, build cb74dfc
- GitHub codespaces prebuild

### Bug description

running `rocker_scripts/install_jupyter.sh` after `apt update && apt upgrade -y` fails with this error:

```
The downloaded source packages are in
	‘/tmp/downloaded_packages’
> remotes::install_github("IRkernel/IRkernel@*release")
Error in github_resolve_ref.github_release(meta$ref %||% ref, meta, host = host,  : 
  Cannot find repo IRkernel/IRkernel.
cannot open URL 'https://api.github.com/repos/IRkernel/IRkernel/releases'
Calls: <Anonymous> ... github_resolve_ref -> github_resolve_ref.github_release
Execution halted
```

### How to reproduce this bug?

```sh
docker run --rm -it rocker/rstudio:4
root@bc2f50bf1da3:/# apt update && apt upgrade -y && bash rocker_scripts/install_jupyter.sh
```

**Comment by breiter** [comment]
I think this was caused by a regression in a security fix for curl in Ubuntu 22.04 and is now fixed.

> ==========================================================================
> Ubuntu Security Notice USN-6237-2
> July 19, 2023
> 
> curl regression
> ==========================================================================
> 
> A security issue affects these releases of Ubuntu and its derivatives:
> 
> - Ubuntu 22.04 LTS
> 
> Summary:
> 
> USN-6237-1 introduced a regression in curl.
> 
> Software Description:
> - curl: HTTP, HTTPS, and FTP client and client libraries
> 
> Details:
> 
> USN-6237-1 fixed vulnerabilities in curl. The update caused a certificate
> wildcard handling regression on Ubuntu 22.04 LTS. This update fixes the
> problem.
> 
> We apologize for the inconvenience.

**Comment by cboettig** [comment]
yup, thanks for reporting, that was indeed quite annoying! glad it is resolved
